src/EPR.Accreditation.API/Repositories/Repository.cs

```diff
@@ -340,7 +342,7 @@ public async Task<IEnumerable<DTO.Country>> GetCountries()
             if (entity.ExemptionReferences.Any())
                 entity.ExemptionReferences.Clear();
 
-            if (entity.ExemptionReferences != null && entity.ExemptionReferences.Count > 0)
+            if (site.ExemptionReferences != null && site.ExemptionReferences.Count() > 0)
```

Comment:
Make _.Count() > 0_ .Any() instead. It's a better function to use for this sort of thing. It'll return true if it find just one record, whereas count will count the whole list.